spacepy/pybats/__init__.py

```diff
@@ -1721,16 +1747,19 @@ def __init__(self, filename=False, load=True, npoints=0, *args, **kwargs):
 
         # Initialize data object and required attributes.
         super(ImfInput, self).__init__(*args, **kwargs)
-        self.attrs['var']= ['bx', 'by', 'bz', 'ux', 'uy', 'uz', 'rho', 'temp']
-        self.attrs['std_var']=True
-        self.attrs['coor']='GSM'
-        self.attrs['satxyz']=[None, None, None]
-        self.attrs['zerobx']=False
-        self.attrs['reread']=False
-        self.attrs['delay']=None
-        self.attrs['plane']=[None, None]
-        self.attrs['header']=[]
-        self['time']=dmarray(zeros(npoints, dtype=object))
+        self.attrs['var'] = ['bx', 'by', 'bz', 'ux', 'uy', 'uz', 'rho', 't']
```

Comment:
Since temp is being renamed, this should be documented in the class's main docstring (probably with a versionchanged directive).

Reply:
Variable naming fixed in upcoming merge- this was indeed a problem to be fixed.